node/src/effects.rs

```diff
@@ -37,8 +36,6 @@ pub fn effects<S: Service>(store: &mut Store<S>, action: ActionWithMeta) {
             store.dispatch(ExternalSnarkWorkerAction::Start);
 
             if store.state().p2p.ready().is_some() {
-                p2p_timeout_effects(store, &meta);
```

Comment:
@0xMimir what replaces this?

Reply:
This is replaced by `P2pState::p2p_timeout_dispatch` in reducer

Reply:
Ok, I see it in the handler of `Action::CheckTimeouts`